### PR TITLE
NAS-120419 / 23.10 / Improvements for Installed Apps table

### PR DESCRIPTION
**Pull request by bvasilenko**
**Testing**

Test 4 improvements mentioned in the ticket

### CODE REVIEW

File: src/app/pages/apps/components/installed-apps/installed-apps.component.html [hunk 1]
```diff
@@ -44,6 +45,9 @@
               (keydown.enter)="selectApp(app)"
             ></ix-app-row>
           </ng-container>
+          <div *ngIf="!dataSource.length" class="no-apps">
+            {{ 'Kubernetes is not running or does not have installed apps' | translate }}
```
Comment (denysbutenko) [inline src/app/pages/apps/components/installed-apps/installed-apps.component.html:49]: Let's keep the old wording.

- When Kubernetes is not running – `Applications are not running`. 
- When no installed apps - `No Applications Installed`

image
image

File: src/app/pages/apps/components/installed-apps/installed-apps.component.html [hunk 2]
```diff
@@ -18,6 +18,7 @@
           <div>
             <span class="app-name-header">
               <mat-checkbox
+                *ngIf="dataSource.length"
                 color="primary"
                 ix-auto
```
Comment (denysbutenko) [inline src/app/pages/apps/components/installed-apps/installed-apps.component.html:23]: `ix-auto` directive has been removed

File: src/app/pages/apps/components/installed-apps/app-row/app-row.component.html [hunk 3]
```diff
@@ -7,13 +7,26 @@
         ix-auto-type="checkbox"
         [ixTest]="application.name"
         [checked]="application.selected"
+        (click)="$event.stopPropagation()"
         (change)="toggleAppChecked($event.checked)"
       ></mat-checkbox>
+      <img [src]="application.chart_metadata.icon" [src-fallback]="imagePlaceholder" />
```
Comment (denysbutenko) [inline src/app/pages/apps/components/installed-apps/app-row/app-row.component.html:13]: There is [app-card-logo component](https://github.com/truenas/webui/blob/master/src/app/pages/apps/components/app-card-logo/app-card-logo.component.ts#L13), please consider if it is applicable here.

**Review by denysbutenko — CHANGES_REQUESTED** [review]

**Comment by bvasilenko** [comment]
@denysbutenko Ok, should be fixed now

**Review by denysbutenko — APPROVED** [review]
image
image
image

Looks good.

**Pull request merged**